sandbox/grommet-app/src/pages/layouts/kinds/DSCCSystemDetail/DetailPane/Capacity.tsx

```diff
@@ -0,0 +1,127 @@
+  const onToggle = (event: { value?: string | string[] }) => {
+    const nextRange = event.value;
+    const nextData = capacityData.filter((d) => {
+      const date = new Date(d.timestamp);
+      switch (nextRange) {
+        case "1H":
+          return date > timeframes["1H"];
+        case "8H":
+          return date > timeframes["8H"];
+        case "1D":
+          return date > timeframes["1D"];
+        case "1W":
+          return date > timeframes["1W"];
+        default:
+          return false;
+      }
+    });
+    setRange([timeframes[nextRange as keyof typeof timeframes], MOCK_TODAY]);
+    setData(nextData);
+  }
+
+  return (
+    <ContentPane
+      heading={undefined}
+      level={undefined}
+      actions={undefined}
+      skeleton={undefined}
+    >
+      <Data data={capacityData}>
+        <Box gap="medium">
+          <Box gap="xxsmall">
+            <Toolbar>
+              <ToggleGroup
+                a11yTitle="Choose chart timeframe"
+                options={[
+                  { label: "1H", value: "1H" },
+                  { label: "8H", value: "8H" },
+                  { label: "1D", value: "1D" },
+                  { label: "1W", value: "1W" },
+                ]}
+                onToggle={onToggle}
+                defaultValue={["1W"]}
+              />
+            </Toolbar>
+            <Text size="small">
+              {Intl.DateTimeFormat(undefined, { dateStyle: "medium", timeStyle: "medium" }).format(range[0])}
+              {' '}&ndash;{' '}
+              {Intl.DateTimeFormat(undefined, { dateStyle: "medium", timeStyle: "medium" }).format(range[1])}
+            </Text>
+          </Box>
+          <Box height="medium">
+            <DataChart
+              data={data}
+              series={[
+                {
+                  property: "timestamp",
+                  label: "Time",
+                  render: (datum: any) => Intl.DateTimeFormat(undefined, {
+                    dateStyle: "short",
+                    timeStyle: "short"
+                  }).format(new Date(datum))
+                },
+                {
+                  property: "used",
+                  label: "Used",
+                  suffix: " TiB"
+                },
+                {
+                  property: "total capacity",
+                  label: "Total Capacity",
+                  suffix: " TiB"
+                }
+              ]}
+              axis={{
+                x: { property: "timestamp", granularity: "medium" },
+                y: { property: "used", granularity: "fine" },
+              }}
+              chart={[
+                { property: "used", type: "line", thickness: "xxsmall" },
+                { property: "total capacity", type: "line", thickness: "xsmall" },
+              ]}
+              size="fill"
+              guide={{
+                x: true,
+                y: {
+                  granularity: "fine",
+                },
+              }}
+              legend
```

Comment:
Is this a grommet thing? seems odd that the legend indicator is two different widths

<img width="262" alt="Screenshot 2025-02-20 at 10 48 53 AM" src="https://github.com/user-attachments/assets/56855067-5926-4ec5-91de-3f69535a4c79" />